### PR TITLE
Add babel 6 version of the inline-require transform

### PR DESCRIPTION
**Pull request by tadeuzagallo**
Add a new version of the transform that will be used in react-native when upgrading to babel 6. I didn't put the tests in `__tests__` because of the other transforms that are babel 5-only compatible.

### CODE REVIEW

**Comment by zpao** [comment]
This should be fine to ship whenever. Thanks for keeping it independent for the time being, we can move it to overwrite the current one once all of our things are using babel 6.

cc @cpojer for a sanity check

**Comment by cpojer** [comment]
cc @voideanvalue

**Comment by vjeux** [comment]
lgtm

File: scripts/babel-6/inline-requires-test.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  it('should inline single usage', function() {
+    compare([
+      'var foo = require("foo");',
+      'foo.bar()'
+    ], [
+      'require("foo").bar();'
+    ]);
+  });
+
+  it('should inline requires that are not assigned', function() {
```
Comment (voideanvalue) [inline scripts/babel-6/inline-requires-test.js]: Technically, these are just left unaffected.

**Comment by voideanvalue** [comment]
Looks good :). Thanks for adding some tests!

File: scripts/babel-6/inline-requires-test.js (hunk 1 above)
Comment (tadeuzagallo) [inline scripts/babel-6/inline-requires-test.js]: Yea, this was a terrible test name, I just kept going with the `should inline...`, sorry :(

**Pull request merged**